### DISCUSSION TITLE
config for dart, python, kotlin

### DISCUSSION THREAD

**Discussion by BtAbdul** [LazyVim/LazyVim#1370] [General]
i need configuration for dart or python or kotlin.  If you have any of them config please share with me, it would more better if i have all of them in one config,   I'm beginner with lazyvim, making configuration from scratch is pretty hard for me, so i need prepared configuration that would help me to learn those languages

**Comment by dpetka2001** [comment]
There is an extra for Python. Read the [docs](https://www.lazyvim.org/extras/lang/python) for how to enable it. And here are my own Python [dotfiles](https://github.com/dpetka2001/dotfiles/blob/main/dot_config/nvim/lua/plugins/extras/lang/python.lua) which are basically the same, except that I also configure a formatter for Python, which the extra lacks.

**Reply by BtAbdul** [reply]
Thank you
